elements/lisk-utils/src/math/q.ts

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-bitwise */
+/*
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+import { ONE, Q_OPERATION, TWO, ZERO } from './constants';
+
+const numberToQ = (base: bigint, val: number): bigint => {
```

Comment:
`Q.constructor()` and `Q.fromValue()` both use `val` as the first and `base` as the second argument.

I think that `numberToQ()` should be consistent with that convention.